src/components/Synthetics/SelectorBase/SelectorBase.scss

```
@@ -7,7 +7,7 @@
 
     cursor: pointer;
 
-    &:hover:not(&-disabled) {
+    &:hover:not(&-disabled) * {
```

does this still select an element itself?

```suggestion
    &:hover:not(&-disabled), &:hover:not(&-disabled) * {
```